### ISSUE TITLE
Cant connect to my pc

### ISSUE THREAD

**Issue by ladel46** [moonlight-stream/moonlight-android#192]
So basicly i cant connect to my pc for some reason i dont find it and whene i do add and put my ip in it it say unable to connect ...

**Comment by JohnCrist1988** [comment]
Can you be more specific about the error you're receiving? What steps have you taken to try to resolve the issue yourself so far? Is your device and computer on the same network?

**Comment by ladel46** [comment]
i basicly can connect throught external ip adress but cant connect to my local adress 
and it say make sure the required ports are allowed ? 
so which ports do i have to allow ?

**Comment by JohnCrist1988** [comment]
If you're connecting from inside your home network, meaning that both your computer and phone/device are on the same network (Wireless, USB, etc), just press the +, type in the IP address of your desktop, and it'll either update the computer's IP or ask you to re-pair the device with a new PIN. The app was updated a few hours ago, so be sure you have the most updated version of the App and make sure the GeForce Experience software is also fully up to date. 

What version of Windows are you running?

**Comment by ladel46** [comment]
i had did that already but it say unable to connect to this spesifique computer im using win10

**Comment by ShawNMusliM** [comment]
I too am unable to connect to my PC. I have reinstalled/rebooted both my computer and my android. Reinstalled moonlight. Tried to connect manually with my local IP address. Tried connecting through my external IP address. Made sure my experience was up to date. Made sure I had compatible games. Made sure the box to allow game streaming was checked. Deleted my anti virus and turned off my firewall. 

This just doesnt work for me.
